### PR TITLE
[Communication] - CallingServer - Update SDK docs. 

### PR DESCRIPTION
**Pull request by zihzhan-msft**

### CODE REVIEW

File: sdk/communication/azure-communication-callingserver/pom.xml [hunk 1]
```diff
@@ -54,7 +54,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-communication-common</artifactId>
-      <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
+      <version>1.0.2</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
```
Comment (JianpingChen) [inline sdk/communication/azure-communication-callingserver/pom.xml:57]:
```suggestion
      <version>1.0.2</version> <!-- {x-version-update;com.azure:azure-communication-common;dependency} -->
```

File: sdk/communication/azure-communication-callingserver/README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ This package contains a Java SDK for Azure Communication CallingServer Service.
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-communication-callingserver</artifactId>
-    <version>1.0.0-beta.1</version>
+    <version>1.0.0-beta.2</version>
```
Comment (JianpingChen) [inline sdk/communication/azure-communication-callingserver/README.md:24]: Why do you bump up version here? Was beta.1 released already? If So, modify ChangeLog.md to reflect the version changes.

**Review by JianpingChen — CHANGES_REQUESTED** [review]

File: sdk/communication/azure-communication-callingserver/pom.xml (hunk 1 above)
Comment (zihzhan-msft) [reply sdk/communication/azure-communication-callingserver/pom.xml]: Change common project version back to 1.1.0-beta.1

File: sdk/communication/azure-communication-callingserver/README.md (hunk 2 above)
Comment (zihzhan-msft) [reply sdk/communication/azure-communication-callingserver/README.md:24]: Updated changelog file.

File: sdk/communication/pom.xml [hunk 3]
```diff
@@ -25,7 +25,7 @@
         <dependency>
           <groupId>com.azure</groupId>
           <artifactId>azure-communication-callingserver</artifactId>
-          <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-communication-callingserver;current} -->
+          <version>1.0.0-beta.2</version> <!-- {x-version-update;com.azure:azure-communication-callingserver;current} -->
```
Comment (JianpingChen) [inline sdk/communication/pom.xml:28]: Also change common version below

**Review by JianpingChen — APPROVED** [review]

**Pull request merged**